<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleWidget</class>
 <widget class="QWidget" name="TitleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#TitleWidget{
background-image: url(:/resource/wizard/first_step.png);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="StepWidget" name="firstWidget" native="true"/>
   </item>
   <item>
    <widget class="StepWidget" name="secondWidget" native="true"/>
   </item>
   <item>
    <widget class="StepWidget" name="thirdWidget" native="true"/>
   </item>
   <item>
    <widget class="StepWidget" name="fourthWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StepWidget</class>
   <extends>QWidget</extends>
   <header>step_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
